src/Microsoft.DotNet.ProjectJsonMigration/PackageConstants.cs

```diff
@@ -16,11 +16,18 @@ internal class PackageConstants
         public const string XUnitRunnerPackageName = "xunit.runner.visualstudio";
         public const string XUnitRunnerPackageVersion = "2.2.0-beta4-build1188";
 
+        public static readonly IDictionary<string, string> TestProjectDependencyPackages = new Dictionary<string, string>
```

Comment:
I don't think we should have a separate map. Maybe what we should do is create an IDictionary<string, PackageDependencyInfo> map. The key would be the source package name. The target PackageDependencyInfo would hold both the new name (which may be the same) and the new version. Or null if the package should be removed.

@jgoshi what do you think?